src/Microsoft.AspNetCore.Authentication/Win32.cs

```diff
@@ -3,7 +3,6 @@
 
 
 using System;
-using System.ComponentModel;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.Win32
```

Comment:
BTW, is this file still used? IIRC, it was used by the Twitter provider for cert pinning, but I'm pretty sure it was removed last year. /cc @Tratcher